src/main/java/model/piece/Piece.java

```diff
@@ -4,10 +4,12 @@
 import model.Path;
 import model.Point;
 import model.Team;
+import model.Moving;
 
 public abstract class Piece {
     Team team;
     PieceName pieceName;
+    int score;
```

Comment:
내부 멤버변수는 항상 private 으로 선언해주세요. 추상 클래스라도 예외는 없습니다!
캡슐화를 지켜 주세요 😄 

```suggestion
    private Team team;
    private PieceName pieceName;
    private int score;
```

Reply:
private으로 보호하는게 더 안전해 보이는군요!

Reply:
불가피한 경우 getter를 사용하시고, 가능하면 TDA 원칙에 따라 객체 내부에서 처리하는 방법이 있겠죠?

ex) `getTeam().isRed()` -> `isRedTeam()`